.github/scripts/s390x-ci/self-hosted-builder/fs/usr/bin/entrypoint

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```

Comment:
Why this change is needed?

Reply:
You're correct. Those changes are not needed. I'll update it everywhere to previous version.